src/zero/once.clj

```diff
@@ -31,3 +34,37 @@
   under the hood if necessary."
   []
   (util/bearer-token-header-for @the-cached-credentials))
+
+(defn new-credentials-from-service-account
+  "Generate scoped GoogleCredentials from a service account FILE."
+  [^String file]
+  (when-not (nil? file)
+    (let [scopes ["email" "profile" "openid"]
+          credentials (GoogleCredentials/fromStream (FileInputStream. file))]
+      (.createScoped credentials scopes))))
+
+(defn service-account-per-env
+  "Map of environment to service account path."
+  []
+  (letfn [(get-sa [environment] {(keyword environment)
+                                 (get-in env/stuff [(keyword environment) :project-service-account])})]
+    (into {} (map get-sa env/registered-envs))))
```

Comment:
How do I build with this change?
I'm guessing this will be simpler after merging PR#11.
https://github.com/broadinstitute/wfl/pull/11

Reply:
Yes! #11 simplifies this a lot, thanks!